### PR TITLE
Add parsing JSON payload of request into Request Object. Closes #9

### LINKED ISSUE #9: Add parsing JSON payload of request into Request Object
Right now `http_json` transport is able to process parameters that are given as query or form parameters. It will be useful if `http_json` will parse actual JSON that can come with POST requests.

### PR DESCRIPTION
**Pull request by nordligulv**

### CODE REVIEW

File: handlers_manager.go [hunk 1]
```diff
@@ -127,9 +127,8 @@ func (hm *HandlersManager) RegisterHandler(h IHandler) error {
 
 		hm.handlerVersions[route] = version
 
-		if _, ok := handlerType.MethodByName("V" + strconv.Itoa(v) + "UseCache"); ok {
-			version.UseCache = true
-		}
+		_, version.UseCache = handlerType.MethodByName(handlerMethodPrefix + "UseCache")
+		_, version.AcceptJSON = handlerType.MethodByName(handlerMethodPrefix + "AcceptJSON")
```
Comment (sergei-svistunov) [inline handlers_manager.go]: You should automatically detect can In arguments be sending by URI params or not (flat structure or not). And this logic must be in transport.

File: transport/http_json/http_json.go [hunk 2]
```diff
@@ -104,8 +108,55 @@ func (h *APIHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
...
+		defer req.Body.Close()
+		maxFormSize := int64(10 << 20) // 10 MB is a lot of text.
+		reader := io.LimitReader(req.Body, maxFormSize+1)
+		body, err := ioutil.ReadAll(reader)
+		if err != nil {
+			log.Print(err.Error())
+			http.Error(w, "failed to read body", http.StatusInternalServerError)
+			return
+		}
+		params, paramsErr = h.hm.PrepareJsonParameters(ctx, handler, body)
```
Comment (sergei-svistunov) [inline transport/http_json/http_json.go]: HM must know nothing about implementation in transport.

File: handlers_manager.go (hunk 1 above)
Comment (nordligulv) [inline handlers_manager.go]: done

File: transport/http_json/http_json.go (hunk 2 above)
Comment (nordligulv) [inline transport/http_json/http_json.go]: done

File: transport/http_json/json_parameters_getter.go [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+)
+
+type JsonParametersGetter struct {
+	Req    *http.Request
+	values map[string]interface{}
+}
+
+func (p *JsonParametersGetter) Parse() error {
+	defer p.Req.Body.Close()
+	maxFormSize := int64(10 << 20) // 10 MB is a lot of text.
```
Comment (sergei-svistunov) [inline transport/http_json/json_parameters_getter.go]: It should be variable. Developers must have ability to tune it.

Comment (nordligulv) [inline transport/http_json/json_parameters_getter.go]: done

**Pull request merged**